File: geeksforgeeks/Binary matrix having maximum number of 1s/Solution.java
class Solution {

    public int[] findMaxRow(int mat[][], int N) {

        int rowNum = 0, maxNum = 0;

        int index = N - 1;

        for (int i = 0; i < N; i++) {

            while (index >= 0 && mat[i][index] == 1) {

                index--;

                maxNum++;

                rowNum = i;

            }

        }

        return new int[] {rowNum, maxNum};

    }

};

File: geeksforgeeks/Binary matrix having maximum number of 1s/Solution.py
#User function Template for python3



class Solution:

    def findMaxRow(self, mat, N):

        # Code here

        ans=[] 

        for i in mat:

            ans.append(i.count(1))

        return ans.index(max(ans)),max(ans)





#{ 

 # Driver Code Starts

#Initial Template for Python 3



if __name__ == '__main__':

    t = int(input())

    for _ in range(t):

        n = int(input())

        mat = []

        for i in range(n):

            A = [int(x) for x in input().split()]

            mat.append(A)

        ob=Solution()

        ans = []

        ans = ob.findMaxRow(mat, n)

        for i in range(2):

            print(ans[i], end =" ")

        print()

# } Driver Code Ends
